### ISSUE TITLE
Undo/Redo jumps not visible at times

### ISSUE THREAD

**Issue by loonies** [vrapper/vrapper#627]
This pretty much annoys me. Sometimes (depending on change location)
when you do multiple subsequent undos/redos, page jumps like crazy and
sometimes it doesn't at all (undo/redo is done out of page viewport)
i.e. you don't even see that change was undone/redone.

If undo/redo is done out of viewport, it would be nice to make change
visible by moving the page in the viewport.

**Comment by albertdev** [comment]
I'll file this as an "Annoyance" then.

Do you know of a scenario to reliably reproduce it? I would like to have some test to see the difference before and after I make any changes.

I think the problem is in https://github.com/vrapper/vrapper/blob/master/net.sourceforge.vrapper.eclipse/src/net/sourceforge/vrapper/eclipse/interceptor/CaretPositionUndoHandler.java#L75 so at least I know where to start...

**Comment by loonies** [comment]
Not really, I just ignored it so far. Once it occurs, I'll make a note.

**Comment by loonies** [comment]
It seems to me that on _undo_ command the window is not scrolled
(repositioned) if caret is jumping outside of the current viewport.
